File: BOJ_단계/1차원 배열/10818.cpp
#include<iostream>

using namespace std;

int main() {
    int N;
    int num;
    int max_n = -1000000;
    int min_n = 1000000;

    cin >> N;

    for (int i = 0; i < N; i++) {
        cin >> num;
        if (num >= max_n)
            max_n = num;
        if (num <= min_n)
            min_n = num;
    }

    cout << min_n << " " << max_n;

}

File: BOJ_단계/1차원 배열/10818.py
max_n = -1000000;
min_n = 1000000;

N = int(input())
arr = input().split()

for i in range(N):
    if int(arr[i]) <= min_n:
        min_n = int(arr[i])
    if int(arr[i]) >= max_n:
        max_n = int(arr[i])

print(str(min_n) + " " + str(max_n))